cumulusci/cumulusci.yml

```diff
@@ -1022,6 +1022,36 @@ flows:
             3:
                 task: install_managed_beta
 
+    release_2gp_beta:
+        group: Release Operations
+        description: Upload and release a beta version of the metadata in this repository
+        steps:
+          1:
+            task: create_package_version
+            options:
+              package_type: Managed
+              package_name: $project_config.project__package__name
+              skip_validation: True
```

Comment:
We need to NOT skip validation here, because package versions that skipped validation cannot be promoted.

Reply:
done